Docs/ios-screen-sharing.md

```diff
@@ -44,15 +44,17 @@ flowchart LR
 
 To use the Broadcast Capture mode, follow these steps to add a Broadcast Upload Extension target and associated configuration to your project. You can also refer to the [example app](https://github.com/livekit-examples/swift-example), which demonstrates this configuration.
 
-#### 1. Add Broadcast Upload Extension Target
 
+#### 1. Add Broadcast Upload Extension Target
 
 <img src="Resources/new-target-options.png" width="500" />
 
 1. In Xcode, Choose "File" > "New > "Target"
 2. From the template chooser, select "Broadcast Upload Extension"
-3. Name the extension (e.g. "BroadcastExtension"). Take note of the extension's bundle identifier, as it will be needed later.
-4. Replace the default content of `SampleHandler.swift` in the new target with the following:
+3. Name the extension (e.g. "BroadcastExtension").
+4. Press "Finish"
+5. From the "Signing & Capabilities" tab of the new target, change the bundle identifier to `<BUNDLE_ID>.broadcast`, replacing `<BUNDLE_ID>` with the bundle identifier of your primary app target.
```

Comment:
I'd say "set it's bundle identifier to be the same as your main app, with `.broadcast` added at the end (to use a different identifier, see Custom Identifiers below)"